copter/source/docs/auto-mode.rst

```diff
@@ -22,6 +22,8 @@ Overview
 
 AUTO mode navigates between each waypoint in a straight line and as it approaches each waypoint follows a smooth curving path towards the next waypoint (S-Curve). It controls the path as it approaches a waypoint to be within the WPNAV_RADIUS of each waypoint. The speed is lowered below :ref:`WPNAV_SPEED<WPNAV_SPEED>`, as necessary, to keep the path within the set corner lateral acceleration limit parameter (:ref:`WPNAV_ACCEL_C<WPNAV_ACCEL_C>`).
 
+.. note:: this smooth curving path as it changes direction to the next waypoint will not occur if the next mission item after the waypoint is anything other than a normal waypoint, including camera commands, speed change commands, etc. In this case the copter will approach the waypoint, stop, turn, and proceed to the next navigation waypoint instead of smoothly curving in front of it and proceeding to that next navigation waypoint. Also, small :ref:`WPNAV_RADIUS<WPNAV_RADIUS>` values will require that the curve be very small, effectively emulating the non-S-Curve behavior explained above.
```

Comment:
Thanks for this.  "Do" and "Conditional" commands will not cause the vehicle to stop at the waypoints though.  How about this?

>The vehicle will stop at the waypoint if a delay has been specified or the next navigation command is not a waypoint or spline command.

I suspect that we may instead need a small section on how to increase the corner cutting.

Reply:
In my mission do change speed after a waypoint killed the scurve on that waypt as I recall...will confirm in Sitl when I can

Reply:
I just tested do-change-speed and it seems OK.  Here's the mission
![image](https://github.com/ArduPilot/ardupilot_wiki/assets/1498098/6dae38a5-53b5-4de2-9dca-e87a71d2b425)

.. and here's the resulting path
![image](https://github.com/ArduPilot/ardupilot_wiki/assets/1498098/98f7b503-0fb3-4715-98a5-bf1aeb3ae6a4)

Reply:
fixed